backend/app/tests/utils/test_data.py

```
@@ -0,0 +1,116 @@
+        credential={
+            Provider.OPENAI.value: {
+                "api_key": api_key,
+                "model": "gpt-4",
+                "temperature": 0.7,
+            }
+        },
+    )
+    return creds_data
+
+
+def create_test_credential(db: Session) -> List[Credential]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)